docs/application-connector/10-02-application-gateway-troubleshooting.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Application Gateway troubleshooting
+type: Troubleshooting
+---
+
+In case user calls registered service and receives an error:
+
+- Verify that the call reached Application Gateway.  
+  To do that fetch logs from Application Gateway pod:
+  ```
+  kubectl -n kyma-integration logs -l app={APP_NAME}-application-gateway -c {APP_NAME}-application-gateway
+  ```
+  If the request reached the pod, it should be logged by Application Gateway.
+  
+  If the call is not in the logs, check if Access Service exists.
```

Comment:
Give more context? What's the Access Service? What's its relation to the service? 
You could link to a place in docs that explains what Acc. Svc. is.